Python/Product/PythonTools/PythonTools/Options/PythonExperimentalOptionsControl.resx

```diff
@@ -240,26 +240,20 @@
   <data name="_useVsCodeDebugger.AutoSize" type="System.Boolean, mscorlib">
     <value>True</value>
   </data>
-  <data name="_useVsCodeDebugger.Enabled" type="System.Boolean, mscorlib">
-    <value>False</value>
-  </data>
   <data name="_useVsCodeDebugger.Location" type="System.Drawing.Point, System.Drawing">
     <value>6, 72</value>
   </data>
   <data name="_useVsCodeDebugger.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
     <value>6, 3, 6, 3</value>
   </data>
   <data name="_useVsCodeDebugger.Size" type="System.Drawing.Size, System.Drawing">
-    <value>135, 17</value>
+    <value>116, 17</value>
   </data>
   <data name="_useVsCodeDebugger.TabIndex" type="System.Int32, mscorlib">
     <value>3</value>
   </data>
   <data name="_useVsCodeDebugger.Text" xml:space="preserve">
-    <value>Use VSCode &amp;debugger</value>
-  </data>
-  <data name="_useVsCodeDebugger.Visible" type="System.Boolean, mscorlib">
-    <value>False</value>
+    <value>Use new &amp;debugger</value>
```

Comment:
"New" seems a bit vague, though I didn't like the previous text that much either. Maybe call it "Use experimental debugger"?